src/containers/BooksForm.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const category = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const BooksForms = () => (
+  <>
+    <input />
+    <select>
```

Comment:
Add attributes: name and type, to the input element, it would help you in the next milestone.